src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -1594,15 +1385,7 @@ class SaagieClient {
                 }
             }
             return exportPipeline
-
-        } catch (InvalidUserDataException invalidUserDataException) {
-            throw invalidUserDataException
-        } catch (GradleException stopActionException) {
-            throw stopActionException
-        } catch (Exception exception) {
-            logger.error('Unknown error in getPipelineAndPipelineVersionDetailToExport')
-            throw exception
-        }
+        }, 'Unknown error in getPipelineAndPipelineVersionDetailToExport') as ExportPipeline
```

Comment:
'Unknown error in task getPipelineAndPipelineVersionDetailToExport, function getAllInstancePipelineInformation'

Reply:
Some method are used in multiple task like this one so i can t log this message